src/components/ImportForm/__tests__/submit-utils.spec.ts

```diff
@@ -76,12 +84,14 @@ describe('Submit Utils: createResources', () => {
     expect(createApplicationMock).toHaveBeenCalledTimes(2);
     expect(createIntegrationTestMock).toHaveBeenCalledTimes(2);
     expect(createComponentMock).toHaveBeenCalledTimes(0);
+    expect(getSecretResourceMock).toHaveBeenCalledTimes(0);
     expect(createImageRepositoryMock).toHaveBeenCalledTimes(0);
   });
 
-  it('should not create application but create components', async () => {
+  it('should not create application, create components and not create secret', async () => {
```

Comment:
This wording seems inaccurete. Will it create componenets or not?

Reply:
The description was update to `should not create application but create components without secrets`